ui/src/Products/ProductCard.tsx

```diff
@@ -75,13 +75,15 @@ function ProductCard({
         setEditMenuIsOpened(false);
     }
 
-    useOnLoad(() => {
+    /* eslint-disable */
```

Comment:
Is there a reason we're ignoring the lint warnings vs just adding `registerProductRef, unregisterProductRef, product` to the array?